core/config/docs/docs.go

```diff
@@ -38,12 +38,12 @@ var (
 	//go:embed example-secrets.toml
 	exampleSecrets string
 
-	docsTOML = coreTOML + chainsEVMTOML + chainsCosmosTOML + chainsSolanaTOML + chainsStarknetTOML
+	DocsTOML = CoreDefaultsTOML + chainsEVMTOML + chainsCosmosTOML + chainsSolanaTOML + chainsStarknetTOML
```

Comment:
Is there any way to avoid exporting these vars? Ideally they would be co-located with their usage. IIRC there was an obstacle to using an internal/ directory when I first implemented this, but that may have changed.